cumulusci.yml

```diff
@@ -832,26 +834,32 @@ flows:
         group: NPSP
         steps:
             1:
-                task: execute_anon
-                # Enables RD2 in Custom Settings. Do this before enabling CRLP so it does not create a rollup for NextDonationDate.
-                options:
-                    apex: >
-                        npe03__Recurring_Donations_Settings__c rdSettings = npe03__Recurring_Donations_Settings__c.getOrgDefaults();
-                        rdSettings.%%%NAMESPACE%%%isRecurringDonations2Enabled__c = true;
-                        upsert rdSettings;
-            2:
                 task: enable_crlp
                 # Enable customizable rollups after enabling RD2
-            3:
-                task: deploy_rd2_config
-                # Deploys the unpackaged configuration required for RD2
-            4:
+            2:
                 task: custom_settings_value_wait
                 # Wait for CRLP to be fully enabled before continuing with data migration
                 options:
                     object: "%%%NAMESPACE%%%Customizable_Rollup_Settings__c"
                     field: "%%%NAMESPACE%%%Customizable_Rollups_Enabled__c"
                     value: true
+            3:
+                task: execute_anon
+                # Enables RD2 in Custom Settings.
```

Comment:
@meena-sfdc I changed the order of the operations in the `deploy_rd2_testing` flow because the EnableEnhancedRecurringDonations operation requires and checks for CRLP being enabled. As a result, it was necessary to move the enable crlp step above this step.

As part of this, there's an additional step that will be required in the apex. In the normal enablement process, there is a call to `System.enqueueJob(new EnablementQueueable(QueueableMode.DisableRollup));` in the `RD2_EnablementDelegate_CTRL.launchMetaDeploy()` method. It will be necessary to call that enqueueJob() operation in the `callableApiEnableEnhancement()` method.

This step disables one of the Customizable Rollups that is no longer used in RD2. I'll update the work item accordingly, though it may not be until a little later this morning.

Reply:
@force2b , I have included the enqueue job in the callableApiEnableEnhancement(). Can you check if everything is ok now?